### DISCUSSION TITLE
Hobby plan developer deploying on Pro plan account.

### DISCUSSION THREAD

**Discussion by YashNaj** [vercel/community#4040] [Help]
### Summary

I have noticed since about July of this year that my commits to a private Org repository hang at the deployment step. Usually resolved with a simple redeploy, but still annoying. It's possible that it is a config thing in my repository, and I can provide code if necessary, but I wanted to check to see if its something on the Vercel side first. 

### Steps to Reproduce

- Push a commit to master 
- Build is succesfull
- Deployment step hangs for 30-44 minutes and then aborts

**Comment by amyegan** [comment]
Hey @YashNaj. I'm not aware of a Vercel problem causing the behavior you described. If you share a deployment ID, we can use that to look up more details about it.
